src/Uno.UWP/UI/Input/Preview.Injection/InputInjector.cs

```diff
@@ -15,7 +15,7 @@ public partial class InputInjector
 	[ThreadStatic] private static IInputInjectorTarget? _inputManager;
 
 	internal static void SetTargetForCurrentThread(IInputInjectorTarget manager)
-		=> _inputManager = manager;
+		=> _inputManager ??= manager; // Set only once per thread.
```

Comment:
Should we notify/fail when we're trying to set it to a different instance that the one already set?